.eslintrc.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
     'import/no-extraneous-dependencies': ['error', { devDependencies: ['**/*.test.ts'] }],
     'max-len': [0],
     'import/prefer-default-export': ['off'],
+    'object-curly-newline': ['off'],
```

Comment:
Doesn't this open the door to some funky declarations?

Reply:
The issue was it's erroring on this
```
import { Entity, PrimaryGeneratedColumn, Column, BaseEntity } from 'typeorm';
```
it wants
```
import {
   Entity,
   PrimaryGeneratedColumn,
   Column,
   BaseEntity
} from 'typeorm';
```

Reply:
Oh wait, it looks like it's just a difference of `prettier` vs `eslint`. Yeah I'll fix that

Reply:
Ok, so the issue is that `prettier` will break it into multiline based on line length and `eslint` will break it based on the number of properties. So disabling it for `eslint` means `prettier` still catches it. IMO I prefer `prettier`'s method.